docs/api/migration-guides/v2-primitives.mdx

```diff
@@ -0,0 +1,1202 @@
+---
+title: Migrate to the V2 primitives
+description: Migrate from using the original primitives (Estimator and Sampler) to using the updated V2 primitives (EstimatorV2 and SamplerV2)
+
+---
+
+<span id="v2-primitives"></span>
+# Migrate to the V2 primitives
+
+This guide describes what changed in the V2 primitives and why, describes each new primitive in detail, and gives examples to help you migrate code from using the legacy primitives to the V2 primitives.  The examples in the guide all use the Qiskit Runtime primitives, but in general, the same changes apply to the other primitive implementations.  The functions unique to Qiskit Runtime such as error mitigation remain unique to Qiskit Runtime. 
+
+## Overview
+
+Version 2 of the primitives is introduced with a new base class for both Sampler and Estimator ([BaseSamplerV2](/api/qiskit/qiskit.primitives.BaseSamplerV2) and [BaseEstimatorV2](/api/qiskit/qiskit.primitives.BaseEstimatorV2)), along with new types for their inputs and outputs. The emphasis of this new version is on performing vectorized calls to the `primitive run()` methods so that you can efficiently specify sweeps over parameter value sets and observables. See [StatevectorSampler](/api/qiskit/qiskit.primitives.StatevectorSampler) and [StatevectorEstimator](/api/qiskit/qiskit.primitives.StatevectorEstimator) for V2 primitive reference implementations.
+
+Based on user feedback, this version introduces the following major new functions:
+
+* A new interface that lets you specify a single circuit and multiple observables (if using Estimator) and parameter value sets for that circuit. Previously, you had to specify the  same circuit multiple times to match the size of the data to be combined.
+* SamplerV2 
+  - Returns the outputs (bitstrings) from every shot, rather than providing a counts-like return (quasi-probabilities). The result class, however, has methods to return weighted samples, such as counts.
+  - Returns data from separate `ClassicalRegisters`. This enables derived classes to implement Sampler support for circuits with classical control flows.
+  - No longer supports resilience levels (error mitigation).  
+* EstimatorV2 
+  - Has a new `precision` argument in the `run()` method that specifies the targeted precision of the expectation value estimates.
```

Comment:
It would be good to mention that this replaces `shots`, and again, what do we gain from it (it's a more accurate description of what the parameter is used for? shots don't make a lot of sense in an error-mitigated estimation context? again, I can think about suggestions later)